lib/net/ber/ber_parser.rb

```diff
@@ -10,7 +10,8 @@ module Net::BER::BERParser
     5 => :null,
     6 => :oid,
     10 => :integer,
-    13 => :string # (relative OID)
+    13 => :string, # (relative OID)
+    139 => :array
```

Comment:
Any relevant docs for this `139` value? (Cursory search doesn't bring up anything.)